src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -2785,6 +2785,22 @@ public virtual bool TryGetUri(object entity, out Uri identity)
             return identity != null;
         }
 
+        /// <summary>
+        /// Processes bulk update requests
+        /// </summary>
+        /// <typeparam name="T">The type of top-level objects to be deep updated.</typeparam>
+        /// <param name="objects">The top-level objects of the type to be deep updated.</param>
+        internal virtual void DeepUpdate<T>(params T[] objects)

```

Comment:
What did you think about the suggestion to name this `BulkUpdate` since that's the term that's being used more - `SaveChangesoptions.BulkUpdate`, `BulkUpdateSaveResult`, `BulkUpdateGraph`, etc?

Reply:
I can name it that. It is just that the term that is mostly used in the spec is `DeepUpdate` and since this is what people will be using to interact with the feature it might be a bit confusing or different.

Reply:
Have named it `BulkUpdate`